README.md

```diff
@@ -14,21 +14,22 @@ All images support the installation of extra features. The following container i
 | native-image | A size compact GraalVM Community container image with the [Native Image](https://www.graalvm.org/reference-manual/native-image) support.                   | RPM-based<sup>1</sup> |
 | truffleruby  | A size compact GraalVM Community container image with the Ruby runtime. It uses a standalone build of [TruffleRuby](https://github.com/oracle/truffleruby/releases). | RPM-based<sup>1</sup> |
 | nodejs       | A size compact GraalVM Community container image with the [Node.js runtime](https://www.graalvm.org/reference-manual/js/NodeJS/).                          | RPM-based<sup>1</sup> |
+| graalpy       | A size compact GraalVM Community container image with the [GraalPy runtime](https://www.graalvm.org/22.2/reference-manual/python/).                          | RPM-based<sup>1</sup> |
```

Comment:
Please use the link without a version: `https://www.graalvm.org/reference-manual/python/`
The empty version URL will always redirect to the latest feature release, for example, `https://www.graalvm.org/reference-manual/python/` will go to `https://www.graalvm.org/22.3/reference-manual/python/`.

Reply:
Thanks for the explanation